e2e-tests/README.md

```diff
@@ -0,0 +1,31 @@
+End to End tests
+================
+
+This section contains the Happy path E2E tests for various flows run using Okta sample apps
+
+## Overview
+
+This directory contains protractor based E2E tests that will be used to test the Okta sample applications
+The tests are divided into the following sections - 
+
+#### Okta-Hosted Login
+This tests the E2E flow for user login that uses the hosted login page on your Okta org
+
+#### Custom Login Page
+This tests the E2E flow for user login that uses the Okta Sign-In Widget on a custom login page 
+
+#### Resource Server
+This tests the E2E flow for authenticating requests (using access tokens issued by Okta) against a resource server
+
+## How to run the tests
+The sub-directories for each scenario contain a protractor configuration file (conf.js) which runs the specs for each scenario.
+
+E.g To run the Okta-Hosted Login tests, run the following command 
```

Comment:
nit: We should add a note here about installing `node_modules` in order to successfully run the protractor tests.

Reply:
Since the tck itself doesn't have a `package.json`, I don't think we need to talk about `npm install`. I've added the following note - It is the responsibility of the sample apps that run these tests to install the required dependencies & setup the environment before running the tests.

Having said that, I have added the dependencies that need to be installed in order to run the protractor tests.